src/javascript_bindings.pyx

```diff
@@ -29,14 +29,15 @@ cdef class JavascriptBindings:
     cpdef py_void SetFunction(self, py_string name, object func):
         self.SetProperty(name, func)
 
-    cpdef py_void SetObject(self, py_string name, object obj):
+    cpdef py_void SetObject(self, py_string name, object obj,
+                            py_bool allow_properties=False):
```

Comment:
Since SetObject can now be used to store not only methods, but also properties, the JavascriptBindings.GetObjectMethod() should be modified and it should be checked whether the value is a method, possibly with code like this: `return callable(self.objects[objectName][methodName])`?